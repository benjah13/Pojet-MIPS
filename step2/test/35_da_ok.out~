Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

Command to execute : NOP 

