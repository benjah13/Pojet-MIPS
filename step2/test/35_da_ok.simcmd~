# test da ok 

da ae1:20
