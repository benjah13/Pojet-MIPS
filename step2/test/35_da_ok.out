00000ae0:	NOP
00000ae4:	NOP
